.env.example

```diff
@@ -59,6 +59,11 @@ SAML_CLIENT_SECRET_VERIFIER=
 # PGSSLMODE='no-verify'
 PGSSLMODE=
 
+# Define which hostnames are expected for the app to work on
+ALLOWED_HOSTNAMES='"cal.com","cal.dev","cal-staging.com","cal.community","cal.local:3000","localhost:3000"'
+#  Reserved orgs subdomains for our own usage
+RESERVED_SUBDOMAINS='"app","auth","docs","design","console","go","status","api","saml","www","matrix","developer","cal","my","team","support","security","blog","learn","admin"'
```

Comment:
Just to confirm, both of this have been added to the deployment variables and have adequate values for us? e.g. `app.mento.co` is the app, and it seems to be referenced here
Also on `ALLOWED_HOSTNAMES` we'd need `cal.mento.co` correct?